db/migrate/20250930132614_change_adjustments_amount_precision.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeAdjustmentsAmountPrecision < ActiveRecord::Migration[7.2]
+  def up
+    safety_assured do # this will lock the table whilst updating the column
```

Comment:
should be fine locking the adjustments table - we have relatively few, and it only affects admin pages.